src/bflib_vidsurface.c

```diff
@@ -106,7 +106,7 @@ TbResult LbScreenSurfaceBlit(struct SSurface *surf, unsigned long x, unsigned lo
 
     if ((blflags & 0x04) != 0) {
```

Comment:
Is there are some enum with that 0x04?

Reply:
Nope, it's all over the place:
https://github.com/Loobinex/keeperfx-unofficial/blob/9e12c67646a4874306f057af65fa8fbed9f6019a/src/bflib_vidsurface.c#L100-L120

Reply:
ok, but it would be best if you grep codebase and create such enum.

Reply:
I plan to clean `bflib_video.c` anyway, right now it's a bit messy, so I can do this alongside.